### PR TITLE
Extra merge branch which applies changes from "Feature/NUM-128_Virologischer-Befund"

### PR DESCRIPTION
**Pull request by NormanWeinert**
Adds the converter for the "Virologischer Befund" template.

Resolves: ehrbase/num_platform#128

Extra merge branch from "Feature/NUM-128_Virologischer-Befund" as there were commit history problems.

If all checks pass it should be ready to merge.

### CODE REVIEW

File: src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerbefund/ProAnalytZugehoerigeLaborprobeChoiceConverter.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+        ProAnalytZugehoerigeLaborprobeDvIdentifier proAnalytZugehoerigeLaborprobeDvIdentifier = new ProAnalytZugehoerigeLaborprobeDvIdentifier();
+
+        mapDvIdentifier(observation).ifPresent(proAnalytZugehoerigeLaborprobeDvIdentifier::setZugehoerigeLaborprobe);
+
+        return proAnalytZugehoerigeLaborprobeDvIdentifier;
+
+    }
+
+    private Optional<DvIdentifier> mapDvIdentifier(Observation observation){
```
Comment (SevKohler) [inline src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerbefund/ProAnalytZugehoerigeLaborprobeChoiceConverter.java]: Add this to DvIdentifierParser as mentioned in the last review. We do not want duplicates. Rest is good to go.

**Review by SevKohler — CHANGES_REQUESTED** [review]

File: src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerbefund/ProAnalytZugehoerigeLaborprobeChoiceConverter.java (hunk 1 above)
Comment (NormanWeinert) [reply src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerbefund/ProAnalytZugehoerigeLaborprobeChoiceConverter.java]: I moved all my parsing methods to DvIdentifierParser.java. Sadly I cannot simply modify and use the existing ones as there an Identifier is expected as an Input and I have an Observation. My main method is therefore now called "parseObservationIntoDvIdentifier" to match the old "parseIdentifierIntoDvIdentifier" in naming style.

DvIdentifierParser.java:
https://github.com/ehrbase/fhir-bridge/pull/417/files#diff-2394ecdc3d20fc7396844daa66a5a87c35e9f5a2386f87e988fb1cef9831a7b0

ProAnalytZugehoerigeLaborprobeChoiceConverter.java:
https://github.com/ehrbase/fhir-bridge/pull/417/files#diff-585a1cf0f329e2ab64afa38a03d4bfe341376a4b2b2937848f1326f68da104f9

**Comment by NormanWeinert** [comment]
@SevKohler Changes were applied and tests ran through again. Everything should be ready now.

File: src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerbefund/ProAnalytZugehoerigeLaborprobeChoiceConverter.java (hunk 1 above)
Comment (SevKohler) [reply src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerbefund/ProAnalytZugehoerigeLaborprobeChoiceConverter.java]: True one should use reflections here, but just leave it be.

**Pull request merged**